Source/WebKit/UIProcess/API/Cocoa/_WKElementAction.mm

```diff
@@ -160,6 +161,14 @@ + (instancetype)_elementActionWithType:(_WKElementActionType)type customTitle:(N
             [assistant handleElementActionWithType:type element:actionInfo needsInteraction:YES];
         };
         break;
+    case _WKElementActionTypeViewSpatial:
```

Comment:
we have both styles in this file, but since this is brand new I'd err on the side of having everything behind the flag (including the case)

Reply:
I can in this case, but in the other two switches in this file where `_WKElementActionTypeViewSpatial` appears I have to leave the case outside the flag or EWS yells at me for not handling all enumeration cases.